debian-setup.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+# Installs all of the necessary packages in a Docker container
+# MUST BE RUN WITH ROOT
+
+apt-get update
+apt-get upgrade -y
+apt-get install -y bc binutils binutils-aarch64-linux-gnu binutils-arm-linux-gnueabi bison ccache curl flex expect gcc git gnupg libssl-dev openssl make qemu-system-arm qemu-system-x86
+curl https://apt.llvm.org/llvm-snapshot.gpg.key | apt-key add -
```

Comment:
I guess it makes sense to have this separate, but if we did this earlier, we'd only have to update once.  I think it's ok to combine them. What do you think?

Reply:
I don't think `curl` is installed by default so I'd need to install it for that command then install everything else. I can add a comment explaining why these two things are separate.

Reply:
Addressed in 18515a3d90b68c06da043f1459e01e45670afe22